ui/pom.xml

```diff
@@ -28,8 +28,8 @@
   <artifactId>integration-azure-oauth-ui</artifactId>
   <packaging>xar</packaging>
 
-  <name>Microsoft Entra ID OpenID Connect (OIDC)</name>
-  <description>This extension allows users to authenticate to the wiki using Entra ID. The app can be purchased individually or part of the XWiki Pro package. Try it free.</description>
+  <name>Microsoft Entra ID SSO</name>
```

Comment:
This naming should be applied for all modules (besides what is specific to the module)